### DISCUSSION TITLE
Failed to resolve: com.haishinkit:haishinkit:0.10.1

### DISCUSSION THREAD

**Discussion by Ajithkumar2** [shogo4405/HaishinKit.kt#85] [Q&A] [answered]
Hi, I'm new to android, can anyone help to resolve this

build.gradle(project)
```
repositories {
        google()
        mavenCentral()
        maven {
            url = uri("https://maven.pkg.github.com/shogo4405/HaishinKit.kt")
            credentials {
                username = System.getenv("GITHUB_USER")
                password = System.getenv("GITHUB_API_TOKEN")
            }
        }
    }
```

build.gradle(app)
```
dependencies {  
 implementation 'com.haishinkit:haishinkit:0.10.1'
 implementation 'com.haishinkit:vulkan:0.10.1'
}
```

**Comment by shogo4405** [comment, marked as answer]
How about JitPack?
```
allprojects {
  repositories {
    maven { url 'https://jitpack.io' }
  }
}
```
```
dependencies {
  implementation 'com.github.shogo4405:HaishinKit~kt:haishinkit:0.10.1'
  implementation 'com.github.shogo4405:HaishinKit~kt:vulkan:0.10.1'
}
```

**Reply by h-rlpl** [reply]
@shogo4405 I tried the jitpack way & faced this isue
![image](https://user-images.githubusercontent.com/53028812/179352061-c0ce83a0-4284-43b8-a501-ee53034962c9.png)

**Reply by shogo4405** [reply]
Can I see your current gradle file?

**Reply by h-rlpl** [reply]
### Project Gradle
// Top-level build file where you can add configuration options common to all sub-projects/modules.
buildscript {
    ext.kotlin_version = '1.7.10'
    repositories {
        google()
        mavenCentral()
    }
    dependencies {
        classpath "com.android.tools.build:gradle:7.0.3"
        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"

        // NOTE: Do not place your application dependencies here; they belong
        // in the individual module build.gradle files
    }
}
allprojects {
    repositories {
        google()
        mavenCentral()
        maven { url 'https://jitpack.io'}
    }
}

task clean(type: Delete) {
    delete rootProject.buildDir
}

### App Gradle
plugins {
    id 'com.android.application'
    id 'kotlin-android'
}

android {
    compileSdk 31

    defaultConfig {
        applicationId "com.example.demo"
        minSdk 26
        targetSdk 31
        versionCode 1
        versionName "1.0"

        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
    }

    buildTypes {
        release {
            minifyEnabled false
            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
        }
    }
    compileOptions {
        sourceCompatibility JavaVersion.VERSION_1_8
        targetCompatibility JavaVersion.VERSION_1_8
    }
}

dependencies {

    implementation 'androidx.appcompat:appcompat:1.4.2'
    implementation 'com.google.android.material:material:1.6.1'
    implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
    implementation 'androidx.legacy:legacy-support-v4:1.0.0'

    testImplementation 'junit:junit:4.+'
    androidTestImplementation 'androidx.test.ext:junit:1.1.3'
    androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'

    implementation 'com.squareup.retrofit2:retrofit:2.9.0'
    implementation 'com.squareup.retrofit2:converter-gson:2.9.0'
    implementation 'androidx.lifecycle:lifecycle-extensions:2.2.0'
    implementation 'com.squareup.okhttp3:logging-interceptor:5.0.0-alpha.2'
    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.4")
    implementation 'com.github.shogo4405.HaishinKit.kt:haishinkit:0.10.1'
    //implementation 'com.github.shogo4405:HaishinKit~kt:haishinkit:0.10.1'
    //implementation 'com.github.shogo4405:HaishinKit~kt:vulkan:0.10.1'

}

**Reply by shogo4405** [reply]
Hi. Maven don't support HaishinKit`.`kt. Pelase use `HaishinKit~kt`.
`implementation 'com.github.shogo4405:HaishinKit~kt:haishinkit:0.10.1'`
- seealso: https://jitpack.io/#shogo4405/HaishinKit~kt/0.10.1

**Reply by h-rlpl** [reply]
Thanks for your input. 
First i was doing HaishinKit~kt. As it didn't work I commented them(last two lines) and was trying with HaishinKit.kt
Neither of them worked.
